vector/src/main/java/im/vector/app/features/settings/devices/v2/list/OtherSessionsController.kt

```diff
@@ -42,18 +44,26 @@ class OtherSessionsController @Inject constructor(
         } else {
             data.take(NUMBER_OF_OTHER_DEVICES_TO_RENDER).forEach { device ->
                 val formattedLastActivityDate = host.dateFormatter.format(device.deviceInfo.lastSeenTs, DateFormatKind.DEFAULT_DATE_AND_TIME)
```

Comment:
To format the date when inactive, I think we should use `DateFormatKind.TIMELINE_DAY_DIVIDER` to only get the date.

Reply:
Fixed in [fbff8d6](https://github.com/vector-im/element-android/pull/6965/commits/fbff8d6408984db70e5a704e30fde34bbc0cc725)